docs/stdlib/json.rst

```diff
@@ -312,12 +308,14 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
     :index: safe navigation
 
-    Return the JSON value at the end of the specified path or an empty set.
+    Returns the value of a :eql:type:`json` at the end of the specified path.
+
+    This will otherwise provide an empty set containing no JSON.
 
     This function provides "safe" navigation of a JSON value. If the
     input path is a valid path for the input JSON object/array, the
     JSON value at the end of that path is returned. If the path cannot
-    be followed for any reason, the empty set is returned.
+    be followed for any reason, the empty set is returned:
```

Comment:
Move the last sentence of this paragraph ("If the path cannot be followed...") to the following paragraph (the one after the code block below). That way, this paragraph talks exclusively about the case where a path can be followed and shows an example of it, while the next paragraph talks exclusively about the case where a path _cannot_ be followed and shows an example of that.